src/components/ImageGrid/ImageGrid.vue

```diff
@@ -6,14 +6,14 @@
     <h5 v-if="isError" class="image-grid__notification py-4">
       {{ fetchState.fetchingError }}
     </h5>
-    <LoadMoreButton
-      v-if="canLoadMore"
-      :is-error="isError"
-      :is-fetching="fetchState.isFetching"
-      :is-finished="fetchState.isFinished"
-      data-testid="load-more"
-      @onLoadMore="onLoadMore"
-    />
+    <footer class="px-2">
+      <VLoadMore
+        v-if="canLoadMore && !fetchState.isFinished"
+        :is-fetching="fetchState.isFetching"
+        data-testid="load-more"
```

Comment:
Instead of a test ID data field should we not give it a very clear ARIA label and use that in the tests? Designing the component around the tests does not seem ideal.

Reply:
I'm going to make a separate issue for search grid accessibility, it should very likely be an aria live region which is fairly involved, could you explain what you mean by "Designing the component around the tests"? Are you opposed to the idea of `data-testid` usage in general?

Reply:
Yeah, my main concern was the use of `data-testid`. We can definitely work it out in a separate PR that improves the accessibility of the button.